modules/graph/src/Workspace.ts

```diff
@@ -208,6 +208,7 @@ export interface PackageJson {
 	os?: Array<string>;
 	// Custom
 	alias?: Array<string>;
+	packageManager?: string;
```

Comment:
Where does this value get initialized?

Reply:
when you use yarn or pnpm, they set this for you